hypha/apply/users/views.py

```diff
@@ -106,9 +107,10 @@ def post(self, *args, **kwargs):
 
 
 @method_decorator(login_required, name="dispatch")
-class AccountView(UpdateView):
+class AccountView(SuccessMessageMixin, UpdateView):
     form_class = ProfileForm
     template_name = "users/account.html"
+    success_message = _("Profile Updated.")
```

Comment:
Very minor but we avoid title case on Hypha. Make it "Profile updated." instead.